api/processors/group.py

```diff
@@ -52,18 +59,55 @@ def _create_log_events_start(cls, group: Group) -> None:
             for teacher in group.teachers.iterator()
         )
 
+    @classmethod
+    def _create_log_events_abort(cls, group: Group) -> None:
+        GroupLogEvent.objects.create(group=group, type=GroupLogEventType.ABORTED)
+
+        CoordinatorLogEvent.objects.bulk_create(
+            CoordinatorLogEvent(
+                coordinator=coordinator,
+                group=group,
+                type=CoordinatorLogEventType.TOOK_NEW_GROUP,
```

Comment:
Shouldn't it also be GROUP_ABORTED?

Reply:
oops, good spot, got too enthusiastic copy-pasting 😄

Reply:
I wonder why the tests didn't catch that 🤔

Reply:
right, just tried the classic `print("here")` trick and it looks like either the test setup or the logic itself is broken - the `coordinators_former` list is not updating. Investigating now...